modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -55,8 +55,62 @@ Manage the API specifications linked to Exchange in your project from the API sp
 [NOTE]
 If you want Studio to list your API when searching in Exchange, you must mark your API specification published as an asset as *stable*.
 
+== Implement AsyncAPI Specifications
+
+Studio supports the implementation of AsyncAPI 2.6 specs. When creating an implementation project, Studio imports and scaffolds an AsyncAPI spec hosted on Exchange into an API interface that you can implement.
```

Comment:
Don't we also support 2.0 specs? @Crispy-Salesforce

Reply:
No, Studio -at least- does not support AsyncAPI 2.0. If you try to scaffold an AsyncAPI 2.0 spec Studio displays an error.